test/e2e/egressip.go

```diff
@@ -1059,6 +1086,222 @@ spec:
+		externalServerContainer, err = providerCtx.CreateExternalContainer(externalServerContainer)
+		framework.ExpectNoError(err, "failed to create external container (%s)", externalServerContainer)
+
+		framework.Logf("Start external containers iperf daemon")
+		logFilePath := "/tmp/iperf-results.json"
+		out, err := infraprovider.Get().ExecExternalContainerCommand(externalServerContainer, []string{
+			"iperf3",
+			"--daemon",
+			"--server",
+			"--verbose",
+			"--port", externalServerContainerPortStr,
+			"--logfile", logFilePath,
+		})
+		framework.ExpectNoError(err, "must start iperf daemon on external container, stdout/stderr:\n%q", out)
+
+		ginkgo.By("2. Create an EgressIP object1 with a single egress IP")
+		var egressIPIP net.IP
+		var externalContainerIP string
+		if utilnet.IsIPv6String(egress1Node.nodeIP) {
+			egressIPIP, err = ipalloc.NewPrimaryIPv6()
+			externalContainerIP = externalServerContainer.GetIPv6()
+		} else {
+			egressIPIP, err = ipalloc.NewPrimaryIPv4()
+			externalContainerIP = externalServerContainer.GetIPv4()
+		}
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred(), "must allocate new IP Node IP")
+		gomega.Expect(net.ParseIP(externalContainerIP)).ShouldNot(gomega.BeNil())
+
+		var egressIPConfig = `apiVersion: k8s.ovn.org/v1
+kind: EgressIP
+metadata:
+    name: ` + egressIPName + `
+spec:
+    egressIPs:
+    - ` + egressIPIP.String() + `
+    podSelector:
+        matchLabels:
+            wants: egress
+    namespaceSelector:
+        matchLabels:
+            name: ` + f.Namespace.Name + `
+`
+		if err := os.WriteFile(egressIPYaml, []byte(egressIPConfig), 0644); err != nil {
+			framework.Failf("Unable to write CRD config to disk: %v", err)
+		}
+		defer func() {
+			if err := os.Remove(egressIPYaml); err != nil {
+				framework.Logf("Unable to remove the CRD config from disk: %v", err)
+			}
+		}()
+
+		framework.Logf("Create the EgressIP configuration")
+		e2ekubectl.RunKubectlOrDie("default", "create", "-f", egressIPYaml)
+
+		ginkgo.By("3. Check that the status is of length one and that one of them is assigned to the correct node")
+		statuses := verifyEgressIPStatusLengthEquals(1, nil)
+		assignedEIPNode := statuses[0].Node
+		// unusedEIPNode is the node that is egress-able but currently no EgressIP is assigned to it. The pod will be
+		// created on this node.
+		unusedEIPNode := egress1Node.name
+		if assignedEIPNode == egress1Node.name {
+			unusedEIPNode = egress2Node.name
+		}
+		framework.Logf("EgressIP %s is assigned to Node %q", egressIPIP.String(), assignedEIPNode)
+
+		framework.Logf("Get baseline network performance with no disruption")
+		clientPodName := "iperfclient-baseline-" + f.Namespace.Name
+		clientPod, err := createPod(f, clientPodName, unusedEIPNode, f.Namespace.Name, []string{"iperf3",
+			"--client", externalContainerIP,
+			"--port", externalServerContainerPortStr,
+			"--time", "2",
+			"--interval", ".1",
+			"--bitrate", "50M",
+			"--parallel", "1",
+			"--json",
```

Comment:
Shall we include 'server push' in this case as I understand that the difference is only adding additional parameter '--reverse' for this client command?